### ISSUE TITLE
Retraining on own dataset

### ISSUE THREAD

**Issue by duvi86** [mateuszbuda/brain-segmentation-pytorch#17]
Hi,

I retrained your model on my own dataset; However, I see the  Dice Loss is 1 because the prediction is totally wrong and the segmented area is really small. During the learning, it does not really change, kind of saturation of DICE. Would you have an idea at first of what is causing that?

**Comment by mateuszbuda** [comment]
Here are some ideas:
- Dice loss of 1 indicates no overlap between GT and predictions. Did you observe it on the training or validation set?
- Dice loss is supposed to be robust on highly imbalanced data but it would be still beneficial to sample data in a way that reduces imbalance (by sampling examples and/or providing crops with a larger GT mask area).
- I recommend you to play with the initial learning rate (maybe it is too high and causes model weights to end up in the area from which it is not able to recover)
- Try reducing the number of filters in the first layer
- If your problem is from a different domain, don't load the model with trained weights from the brain MRI dataset
- Debug you data loading pipeline and make sure that the input to the model and GT masks are correct
- Turn off data augmentation for testing if you have any
